axelor-account/src/main/resources/views/MoveLine.xml

```diff
@@ -659,7 +659,7 @@
       <panel name="invoiceTermFromInvoicePanel" title="Term(s)"
         showIf="account.hasInvoiceTerm  &amp;&amp; move.invoice != null" colSpan="12">
         <button name="showInvoiceTermsFromInvoiceBtn" title="Show related invoice term(s)"
-          onClick="save,action-move-line-view-show-related-invoice-terms-from-invoice"
+          onClick="save,action-move-line-view-show-related-invoice-terms-from-invoice,close"
```

Comment:
This change does not cause issue when we call this action from move line menu form ?

Reply:
Create an issue on move-line-move-fom and can create confusion for user, reverted